src/tiled/tileseteditor.cpp

```diff
@@ -196,7 +196,13 @@ void TilesetEditor::addDocument(Document *document)
 
     TilesetView *view = new TilesetView(mWidgetStack);
     view->setTilesetDocument(tilesetDocument);
-    view->setZoomable(new Zoomable(this));
+
+    auto zoomable = new Zoomable(this);
+    auto fileName = tilesetDocument->fileName();
+    qreal zoom = Preferences::instance()->TilesetScale(fileName);
```

Comment:
hmm, it is exactly the problem i try to solve now. 

If to store two different zooms, you need to add path parameter to `Preferences::tileSetScale` and call it from `TilesetDock` or `TilesetEditor`.

However, using the same zoom value not what the user expects, i think.